From: =?utf-8?q?F=C3=A9lix_Sipma?= <felix.sipma@no-log.org>
Date: Tue, 18 Oct 2016 19:55:44 +0200
Subject: dropTok type fix

---
 src/Patat/Theme.hs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Patat/Theme.hs b/src/Patat/Theme.hs
index c46c7f5..6ace99c 100644
--- a/src/Patat/Theme.hs
+++ b/src/Patat/Theme.hs
@@ -255,6 +255,7 @@ nameForTokenType =
     unCapitalize (x : xs) = toLower x : xs
     unCapitalize xs       = xs
 
+    dropTok :: String -> String
     dropTok str
         | "Tok" `isSuffixOf` str = take (length str - 3) str
         | otherwise              = str
