From: =?utf-8?q?F=C3=A9lix_Sipma?= <felix.sipma@no-log.org>
Date: Fri, 2 Dec 2016 12:37:39 +0100
Subject: use utf8 in extra/make-man.hs

---
 extra/make-man.hs | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/extra/make-man.hs b/extra/make-man.hs
index bb29131..78c01f8 100644
--- a/extra/make-man.hs
+++ b/extra/make-man.hs
@@ -5,6 +5,7 @@ import           Control.Monad       (guard)
 import           Data.Char           (isSpace, toLower)
 import           Data.List           (isPrefixOf)
 import           Data.Maybe          (isJust)
+import qualified GHC.IO.Encoding     as Encoding
 import           System.Environment  (getEnv)
 import qualified System.IO           as IO
 import qualified Text.Pandoc         as Pandoc
@@ -80,6 +81,7 @@ reorganizeSections (Pandoc.Pandoc meta0 blocks0) =
 
 main :: IO ()
 main = do
+    Encoding.setLocaleEncoding Encoding.utf8
     Right pandoc0  <- Pandoc.readMarkdown Pandoc.def <$> readFile "README.md"
     Right template <- Pandoc.getDefaultTemplate Nothing "man"
 
